### ISSUE TITLE
Seldon Core Serving simple example

### ISSUE THREAD

**Issue by ricpet** [kubeflow/kubeflow#5000]
/kind bug

**What steps did you take and what happened:**
I am following the Simple example available [here](https://www.kubeflow.org/docs/components/serving/seldon/) and when I try to create SeldonDeployment I get the following error:

```
Error from server (InternalError): error when creating "STDIN": Internal error occurred: failed calling webhook "mseldondeployment.kb.io": Post https://seldon-core.kubeflow.svc:443/mutate-machinelearning-seldon-io-v1-seldondeployment?timeout=30s: no service port 443 found for service "seldon-core"
```

Any help? idea?

**Environment:**

**- Kubeflow version:** Kubeflow 1.0
**- Kubernetes platform:** microk8s 1.18/stable
**- Kubernetes version**: Client Version: version.Info{Major:"1", Minor:"18", GitVersion:"v1.18.2", GitCommit:"52c56ce7a8272c798dbc29846288d7cd9fbae032", GitTreeState:"clean", BuildDate:"2020-04-16T11:56:40Z", GoVersion:"go1.13.9", Compiler:"gc", Platform:"linux/amd64"}
Server Version: version.Info{Major:"1", Minor:"18", GitVersion:"v1.18.2", GitCommit:"52c56ce7a8272c798dbc29846288d7cd9fbae032", GitTreeState:"clean", BuildDate:"2020-04-16T11:48:36Z", GoVersion:"go1.13.9", Compiler:"gc", Platform:"linux/amd64"}

**- OS:** NAME="Ubuntu"
VERSION="18.04.4 LTS (Bionic Beaver)"

**Comment by jbottum** [comment]
@cliveseldon Hey Clive - Is there someone at Seldon that might respond to this ?

**Comment by cliveseldon** [comment]
This looks like the seldon manager is not responding. We don't presently handle k8s 1.18 which I see you are running. Can you downgrade your microk8s to at least 1.17?
See: https://github.com/SeldonIO/seldon-core/issues/1678

**Comment by ricpet** [comment]
Correct... it works fine on 1.17
Thanks for the feedback. It could be useful to update the documentation here https://www.kubeflow.org/docs/components/serving/seldon/

**Comment by cliveseldon** [comment]
Yes. I will create a PR.

**Comment by cliveseldon** [comment]
https://github.com/kubeflow/website/pull/1943

**Comment by jlewi** [comment]
Closing this issue because I think it is resolved by kubeflow/website#1943
